examples/node-ts/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "1.0.0",
   "scripts": {
     "start": "ts-node src/index.ts",
+    "build": "node ./build.js",
```

Comment:
Why do we need `build` since project already uses `ts-node`

Reply:
I added a build this way I can bundle everything in a JS file and don’t need to worry about copying over node_modules in the container

Reply:
Oh it is "binary"  👍